tool/csv_writer_gobuster.go

```diff
@@ -0,0 +1,70 @@
+package tool
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+func remove(slice []string, s int) []string {
+	return append(slice[:s], slice[s+1:]...)
+}
+
+func lineSkipper(data *os.File) *os.File {
+	r := bufio.NewReader(data)
+	for {
+		_, err := r.ReadBytes('\n')
+		if err == nil {
+			break
+		}
+	}
+	return data
+}
```

Comment:
Can you explain the goal?

Reply:
If this function was not here it is overwrite the data already writen in the setup of gobuster